testing/unittests/dalitests.cpp

```diff
@@ -1139,7 +1139,15 @@ class CDaliSDSTests : public CppUnit::TestFixture
             }
             ~CSubscriberContainer()
```

Comment:
This also needs noexcept(false) otherwise if an exception is thrown the program will terminate immediately